i6_models/assemblies/conformer/conformer.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+import torch
+from torch import nn
+
+# TODO probably used for SpecAugment
+from torchaudio.transforms import TimeMasking, TimeStretch, FrequencyMasking
+from dataclasses import dataclass
+
+from i6_models.config import ModelConfiguration
+
+from i6_models.parts.conformer.convolution import ConformerConvolutionV1
+from i6_models.parts.conformer.feedforward import ConformerPositionwiseFeedForwardV1
+from i6_models.parts.conformer.mhsa import ConformerMHSAV1
+
+
+@dataclass
+class ConformerFeedForwardV1Config(ModelConfiguration):
+    dim: int
+
+
+@dataclass
+class ConformerMHSAV1Config(ModelConfiguration):
+    pass
+
+
+@dataclass
+class ConformerConvV1Config(ModelConfiguration):
+    pass
```

Comment:
I think it would be nicer to define these in the files where the respective classes are defined rather than here.

Reply:
I agree. They should be added in the corresponding PR imho, I am just adding them here as dummies until they exist.